oauth_client_integration_test.go

```diff
@@ -384,6 +427,140 @@ func TestOAuthClientsCreateOptionsValid(t *testing.T) {
 	})
 }
 
+func TestOAuthClientsAddProjects(t *testing.T) {
+	skipUnlessBeta(t)
```

Comment:
I think it should be possible to remove skipUnlessBeta calls from your new tests because these endpoints are not behind a feature flag and the tests can be run.

Reply:
@Netra2104 It seems like my suggestion was a bad one. It usually takes a day for the test infrastructure to catch up with the platform nightly release. I take it that the 'Remove' endpoint was added today?

Reply:
Yup it was added today. Also the feature is feature flagged: https://github.com/hashicorp/atlas/pull/17752/files#diff-c72e00c021aa8f6d41e04f01728c117228541e94deda71313ed203e936eb5d17R7

Reply:
Ohhhhh sorry about that; I didn't notice.